src/app/component/MissionDetailsCard.jsx

```diff
@@ -0,0 +1,384 @@
+import {
+  Divider,
+  Button,
+  Grid,
+  Box,
+  Avatar,
+  Card,
+  CardHeader,
+  CardContent,
+  CardMedia,
+  CardActions,
+} from "@material-ui/core";
+import { color } from "../../theme";
+import PersonIcon from "@material-ui/icons/Person";
+import LocationOnIcon from "@material-ui/icons/LocationOn";
+import ScheduleIcon from "@material-ui/icons/Schedule";
+import ArrowBackIcon from "@material-ui/icons/ArrowBack";
+import AttachMoneyIcon from "@material-ui/icons/AttachMoney";
+import cameraImage from "../../img/placeholderBackground.svg";
+
+import { makeStyles } from "@material-ui/core/styles";
+import styled from "styled-components";
+
+import { MissionStatus, MissionFundedStatus } from "../model";
+
+const StyledButton = styled(Button)`
+  flex-grow: 1;
+`;
+
+const useStyles = makeStyles((theme) => ({
+  content: {
+    padding: theme.spacing(2),
+  },
+  goBackIcon: {
+    fontSize: 32,
+    fill: color.deepPurple,
+    "&:hover": {
+      cursor: "pointer",
+    },
+  },
+  cardHeader: {
+    paddingBottom: theme.spacing(1),
+  },
+  subheader: {
+    marginTop: theme.spacing(0.3),
+  },
+  contentTypography: {
+    paddingLeft: theme.spacing(0.5),
+  },
+  avatar: {
+    height: "25px",
+    width: "25px",
+    marginBottom: theme.spacing(0.5),
+  },
+  image: {
+    height: 0,
+    paddingTop: "56.25%", // 16:9
+  },
+  missionTypeText: {
+    paddingTop: theme.spacing(0.5),
+  },
+  deliveryDetailsHeader: {
+    paddingTop: theme.spacing(1),
+    fontWeight: 600,
+  },
+  deliveryDetails: {
+    marginTop: theme.spacing(0.5),
+  },
+  unassignButton: {
+    color: color.darkPink,
+    textDecoration: "underline",
+  },
+}));
+
+const titleCase = (str) => ("" + str).charAt(0).toUpperCase() + ("" + str).substr(1);
+
+const MissionDetailsStatus = ({ status, volunteerName }) => {
+  return status === null ? null : status === MissionStatus["unassigned"] ? (
+    titleCase(status)
+  ) : status === MissionStatus["tentative"] || status === MissionStatus["assigned"] ? (
+    volunteerName
+  ) : (
+    <>
+      {volunteerName} - <b>{titleCase(status)}</b>
+    </>
+  );
+};
+
+const MissionDetailsIconList = ({ contentItems, classes, outerClass }) => (
+  <Grid container className={outerClass} alignItems="flex-start">
+    {contentItems.map((contentItem, index) => {
+      const Icon = contentItem.icon;
+      const avatarImage = contentItem.avatar?.image;
+      const content = contentItem.content.map((content, index) => {
+        return (
+          <Body2
+            key={`content-item-txt-${index + 1}`}
+            color="textPrimary"
+            style={content.style}
+            className={classes.contentTypography}
+          >
+            {content.text}
+          </Body2>
+        );
+      });
+
+      return (
+        <React.Fragment key={`content-item-${index + 1}`}>
+          <Grid item xs={1}>
+            {Icon !== undefined ? <Icon color="primary" /> : null}
```

Comment:
Any reasons why you're not using something more readable like the following:
```suggestion
            {Icon && <Icon color="primary" />}
```
?